docs/processing/preprocessing.md

```diff
@@ -18,7 +19,8 @@ We avert that session-wise *fMRIPrep*'s processes run into race conditions by pr
 {% endfilter %}
         ```
 
-- [ ] Once the anatomical workflow ran successfully, submit a *job array* with one scanning session each with the `--bids-filter-file` argument selecting the corresponding session, and point the `--fs-subjects-dir` argument to the folder where *FreeSurfer* results were stored.
+- [ ] Once the anatomical workflow ran successfully, submit a *job array* with one scanning session each with the `--bids-filter-file` argument selecting the
+    corresponding session, and point the `--fs-subjects-dir` argument to the folder where *FreeSurfer* results were stored.
```

Comment:
Our markdown is configured to break lines when you add them, so this will render weirdly (I'm 90% sure of this):
```suggestion
- [ ] Once the anatomical workflow ran successfully, submit a *job array* with one scanning session each with the `--bids-filter-file` argument selecting the corresponding session, and point the `--fs-subjects-dir` argument to the folder where *FreeSurfer* results were stored.
```